features/lorawan/lorastack/mac/LoRaMacCommand.h

```diff
@@ -148,13 +160,16 @@ class LoRaMacCommand {
     bool is_sticky_mac_command_pending();
 
 private:
-    LoRaMac& _lora_mac;
-
     /**
      * Indicates if the MAC layer wants to send MAC commands
      */
     bool mac_cmd_in_next_tx;
 
+    /**
+      * Indicates if there are any pending sticky MAC commands
+      */
+    bool sticky_mac_cmd;
```

Comment:
We now have two similar variables (sticky_mac_cmd vs. mac_cmd_buf_idx_to_repeat). We can later check if these could be combined.

Reply:
This new variable is used in add_mac_command() and merging it with mac_cmd_buf_idx_to_repeat would mean that if the value is non-zero it should be increased but how much, we need to investigate. Plus it might be potential to cause a break in functionality, so we will leave it later as you suggested.